src/datasources/fault-ds/datasource.js

```diff
@@ -40,9 +40,8 @@ export class OpenNMSFMDatasource {
             if (clause.restriction instanceof API.NestedRestriction) {
                 self.substitute(clause.restriction.clauses, options);
             } else if (clause.restriction.value) {
-                // TODO MVR: This is a hint on how to probably best implement HELM-12
-                // clause.restriction.value = clause.restriction.value.replace(/\$timeFrom/g, options.range.from.valueOf());
-                // clause.restriction.value = clause.restriction.value.replace(/\$timeTo/g, options.range.to.valueOf());
+                clause.restriction.value = clause.restriction.value.replace(/\$range_from/g, options.range.from.valueOf());
+                clause.restriction.value = clause.restriction.value.replace(/\$range_to/g, options.range.to.valueOf());
                 clause.restriction.value = self.templateSrv.replace(clause.restriction.value, options.scopedVars);
```

Comment:
Should we not add the `range_from` and `range_to` to the list of `scopedVars`, rather than performing the token replacement ourselves?